spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -209,7 +415,7 @@ definitions:
           desc: Mean motion difference
       - m0:
           type: double
-          units: radians
+          units: rad
```

Comment:
Most of the unit fields use abbreviations, so radians was made to conform to this practice.